Tasks/VsTest/vstest.ts

```diff
@@ -456,22 +704,25 @@ function resetRunInParallel() {
     runInParallel = false;
 }
 
+function compareNumbers(a,b) {
+    return a - b;
```

Comment:
Inbuilt sort function sorts array converting elements to string. Comparer function added to sort based on numbers.